src/Symfony/Component/Messenger/Tests/DependencyInjection/MessengerPassTest.php

```diff
@@ -330,6 +356,14 @@ public function stop(): void
     }
 }
 
+class InvalidDummyReceiver
```

Comment:
You can probably remove `Dummy` in here 🙃

Reply:
Removed.